From: David Faure <faure@kde.org>
Date: Mon, 27 Jul 2015 20:09:38 +0000
Subject: set cmake_min_req to enable newer policies; fix duplicated target name
X-Git-Url: http://quickgit.kde.org/?p=wacomtablet.git&a=commitdiff&h=9b641e5526b8bb5e089d344f4993f1ff77a0a4fe
---
set cmake_min_req to enable newer policies; fix duplicated target name
---


--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 2.8.9)
 
 project(wacomtablet)
 

--- a/autotests/kded/xinputadaptor/CMakeLists.txt
+++ b/autotests/kded/xinputadaptor/CMakeLists.txt
@@ -1,11 +1,11 @@
 
 
-kde4_add_unit_test(KDED.XsetWacomAdaptor
+kde4_add_unit_test(KDED.XinputWacomAdaptor
   testxinputadaptor.cpp
   ../tabletdependenttest.cpp
 )
 
-target_link_libraries(KDED.XsetWacomAdaptor
+target_link_libraries(KDED.XinputWacomAdaptor
 
 # include static libs
         wacom_common

